### PR TITLE
[ci] Add controller test for package :show action

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by mdeniz** [comment]
Beside that I would prefer to not only migrate those two tests and I would prefer instead to cover the `show` action completely too, it LGTM :+1:

**Comment by ChrisBr** [comment]
Discussed with @mdeniz how we can test the before_filter instead

File: src/api/test/functional/webui/package_controller_test.rb [hunk 1]
```diff
@@ -88,11 +88,13 @@ def test_show_package_binary_as_user
     page.must_have_text 'Total build: 503 s'
   end
 
+  # spec/controllers/webui/package_controller_spec.rb
   def test_show_invalid_package
```
Comment (hennevogel) [inline src/api/test/functional/webui/package_controller_test.rb]: Please add the comment after the test name so we can grep more easy for it...

**Comment by hennevogel** [comment]
I like it :bouquet:

**Pull request merged**